### PR TITLE
Fix for #84: xbuild resolver will now attempt to use msbuild if available.

### PR DESCRIPTION
**Pull request by Lansatac**

### CODE REVIEW

**Comment by Lansatac** [comment]
Oops, I was running the wrong tests. I was impressed that it worked first try. I'll amend this when I've got the tests passing.

**Comment by Lansatac** [comment]
It helps to run the msbuild tests rather than the nUnit tests. Let me know if there's any changes I should make.

File: src/main/groovy/com/ullink/PosixMsbuildResolver.groovy [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+class PosixMsbuildResolver implements IExecutableResolver {
+
+    String[] msbuild
+
+    PosixMsbuildResolver(VersionNumber version)
+    {
+        try {
+            msbuild = locateMsBuild(version)
+        }
+        catch (GradleException){}
```
Comment (gluck) [inline src/main/groovy/com/ullink/PosixMsbuildResolver.groovy]: maybe log the exception ?

File: src/test/groovy/com/ullink/PosixMsbuildLocateTests.groovy [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        def msbuildVersion = msbuild[1]
+        p.msbuild {
+            version = msbuildVersion
+        }
+        msbuild = resolver.locateMsBuild(VersionNumber.parse(p.tasks.msbuild.version))
+        assertNotNull(msbuild)
+    }
+
+    @Test
+    public void givenInvalidMSBuildVersion_throwsException() {
```
Comment (gluck) [inline src/test/groovy/com/ullink/PosixMsbuildLocateTests.groovy]: name of the test is no longer applicable, other than that LGTM

Comment (Lansatac) [reply src/test/groovy/com/ullink/PosixMsbuildLocateTests.groovy]: Fixed!

**Pull request merged**